src/main/java/org/recommendation/log/Logger.java

```diff
@@ -0,0 +1,8 @@
+package org.recommendation.log;
+
+public interface Logger {
+    public void info(final String info);
```

Comment:
having only one argument we can have String... to support multiple strings in arguments. I can see you are adding string while calling this method

Reply:
good catch, donr